translator/jsonconfig/mergeJsonConfig.go

```diff
@@ -22,15 +21,14 @@ func MergeJsonConfigMaps(jsonConfigMapMap map[string]map[string]interface{}, def
 		if os.Getenv(config.USE_DEFAULT_CONFIG) == config.USE_DEFAULT_CONFIG_TRUE {
 			// When USE_DEFAULT_CONFIG is true, ECS and EKS will be supposed to use different default config. EKS default config logic will be added when necessary
 			if ecsutil.GetECSUtilSingleton().IsECS() {
-				fmt.Println("No json config files found, use the default ecs config")
+				log.Println("No json config files found, use the default ecs config")
```

Comment:
Are we _sure_ about doing this change? The last time that we switched from fmt to log, it caused a regression.

Reply:
Yes. There is one reason why I introduced this change:
* Even though I have inputed the wrong json format, CWAgent still say `2022/06/24 16:42:59 I! Valid Json input schema.` (under the hood, if there is a wrong json format, CWAgent will use the default one to override it)

This changes only applied to the `config-translator` binaries and after watching the [shell](https://github.com/aws/amazon-cloudwatch-agent/blob/master/packaging/windows/amazon-cloudwatch-agent-ctl.ps1#L377), I believe there won't be any regression error.